src/proxy/index.js

```diff
@@ -29,9 +36,13 @@ const start = async () => {
     }
   });
 
-  proxyApp.listen(proxyHttpPort, () => {
-    console.log(`Proxy Listening on ${proxyHttpPort}`);
+  http.createServer(options, proxyApp).listen(proxyHttpPort, () => {
+    console.log(`HTTP Proxy Listening on ${proxyHttpPort}`);
   });
+  https.createServer(options, proxyApp).listen(proxyHttpsPort, () => {
+    console.log(`HTTPS Proxy Listening on ${proxyHttpsPort}`);
+  });
+
```

Comment:
Can you explain this change? Why has a new dependency been introduced and why are two servers being created, both HTTP and HTTPS?